### PR TITLE
Address validation country area additional mapping

### PR DESCRIPTION
**Pull request by jakubkuc**
I want to merge this change because it resolves issue #12093

The full map is not yet provided. There is only one example for `CH` country area `Graubünden`.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/account/i18n.py [hunk 1]
```diff
@@ -38,6 +38,10 @@
     "zip": "ZIP code",
 }
 
+CountryAreaValues = namedtuple("CountryAreaValues", ["value", "choices"])
+
+COUNTRY_AREA_MAP = {"CH": CountryAreaValues("Graubünden", ["Graubünden"])}
```
Comment (korycins) [inline saleor/account/i18n.py:43]: I may not fully understand the requirements, but do we really need to have and maintain such a map on the core side? It seems like we would have to cover a large number of countries and regions, and manually paste them into the Saleor code. Additionally, we would have to update the list of choices whenever someone reports a missing value, and create a new release with an extended map that would need to be ported to all Saleor versions greater than the requested one.

**Review by IKarbowiak — APPROVED** [review]

File: saleor/account/i18n.py (hunk 1 above)
Comment (maarcingebala) [inline saleor/account/i18n.py:43]: There should be another mapping nested per country, as multiple country areas will exist. The current structure supports only a single country area per country.

**Review by IKarbowiak — APPROVED** [review]

File: saleor/account/i18n.py [hunk 2]
```diff
@@ -38,6 +38,12 @@
     "zip": "ZIP code",
 }
 
+CountryAreaValues = namedtuple("CountryAreaValues", ["value", "choices"])
+
+COUNTRY_AREA_MAP = {"CH": [
```
Comment (8r2y5) [inline saleor/account/i18n.py:43]: Please run black or pre-commit

**Review by 8r2y5 — APPROVED** [review]

**Comment by maarcingebala** [comment]
We still didn't get any specific examples of exceptions that should be included in the map. I think let's close this PR for now and revisit if this is really needed by someone, as for now I believe it's not a priority.

**Pull request closed without merging**